build-test.xml

```diff
@@ -5790,6 +5790,30 @@ tell application "Safari" to quit
 		</sequential>
 	</macrodef>
 
+	<macrodef name="rename-file">
```

Comment:
You can rename a file in ant normally. We shouldn't be using beanshell to rename files as this is just extra work. Ideally we can use a Poshi Utility to rename files but using a normal ant command should be fine also.